appsec-configs/crowdsecurity/generic-rules.yaml

```diff
@@ -0,0 +1,7 @@
+name: crowdsecurity/generic-rules
+default_remediation: ban
+#log_level: debug
+inband_rules:
+ - crowdsecurity/generic-*
+# inband_options:
```

Comment:
i think we can remove commented lines here

Reply:
done